conda_forge_tick/auto_tick.py

```diff
@@ -225,7 +224,13 @@ def run(
                 )
             ]
 
-    if migrator.check_solvable and not is_recipe_solvable(feedstock_dir):
+    if (
+        migrator.check_solvable
```

Comment:
```suggestion
        (
            migrator.check_solvable
            or feedstock_ctx.attrs["conda-forge.yml"]
            .get("bot", {})
            .get("check_solvability", False)
        )
```

Reply:
You should group the or statement together for clarity.

Reply:
Yup you got it!

Reply:
Ok, I've updated that, does it now reflect what you wanted?

Reply:
ignore my suggestion